content/en/workflows/trigger.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Trigger a workflow
+kind: documentation
+disable_toc: false
+further_reading:
+- link: "dashboards"
+  tag: "Documentation"
+  text: "Find out more about setting up a dashboard"
+- link: "security_platform/explorer"
+  tag: "Documentation"
+  text: "Find out more about Security Signals"
+- link: "monitors"
+  tag: "Documentation"
+  text: "Find out more about Monitors"
+---
+
+You can trigger a workflow manually, on a schedule, or using a variety of automated methods such as a Datadog Monitor, or a webhook. Before you can trigger a workflow using automation, you must associate the workflow with a unique service account. For more information on setting up a service account for your workflows, see [Set up workflows][1].
```

Comment:
We no longer support webhooks but we do support security signal notification rules.

Reply:
Ah thanks! I forgot to remove the webhook bit